src/gsi/read_bufrtovs.f90

```diff
@@ -674,22 +732,66 @@ subroutine read_bufrtovs(mype,val_tovs,ithin,isfcalc,&
            endif
 
 !          Read data record.  Increment data counter
-!          TMBR is actually the antenna temperature for most microwave 
-!          sounders.
-           if (llll == 1) then
-              call ufbrep(lnbufr,data1b8,1,nchanl,iret,'TMBR')
-           else     ! EARS / DB
-              call ufbrep(lnbufr,data1b8,1,nchanl,iret,'TMBRST')
-              if ( amsua .or. amsub .or. mhs )then
-                 data1b8x=data1b8
-                 data1b4=data1b8
-                 call remove_antcorr(sc(instrument)%ac,ifov,data1b4)
-                 data1b8=data1b4
-                 do j=1,nchanl
-                    if(data1b8x(j) > r1000)data1b8(j) = 1000000._r_kind
-                 end do
+!          TMBR is actually the antenna temperature for most microwave sounders.
+!          Changed from accepting TMBR and converting TMBRST to
+!          antenna temperature (remove_antcorr), to converting TMBR to
+!          brightness temperature (add_antcorr) and accepting TMBRST
+!
+
+           if (ta2tb) then
+
+              if (llll == 1) then
+                 call ufbrep(lnbufr,data1b8,1,nchanl,iret,'TMBR')
+                 if ( amsua .or. amsub .or. mhs )then
+                    ! convert antenna temperature to brightness temperature
+                    data1b8x=data1b8
+                    data1b4=data1b8
+                    !call apply_antcorr(accoeff_sets(spc_coeff_versions),ifov,data1b4)
+                    call apply_antcorr(accoeff_sets(1),ifov,data1b4)
+                    data1b8=data1b4
+                    do j=1,nchanl
+                       if(data1b8x(j) > r1000) data1b8(j) = 1000000._r_kind
+                    end do
+                 end if
+              else     ! EARS / DB
+                 call ufbrep(lnbufr,data1b8,1,nchanl,iret,'TMBRST')
+                 !if ( amsua .or. amsub .or. mhs .AND. sacv .ne. spc_coeff_versions)then
+                 if ( amsua .or. amsub .or. mhs .AND. spc_coeff_versions /= 1 .AND. sacv .ne. 1)then
```

Comment:
Thank you for replacing the first .ne. with /=.  Please replace the second .ne. with /= on line 759.

Reply:
@MichaelLueken-NOAA Thanks you for your thorough review of my PR.  Your request has been addressed.